aws/aws_test.go

```diff
@@ -104,3 +105,44 @@ func TestAwsGetPrefixFromRootDeviceName(t *testing.T) {
 		assert.Equal(t, test.expectedPrefix, prefix)
 	}
 }
+
+type mockEC2Client struct {
+	ec2iface.EC2API
+	Vol *ec2.Volume
+}
+
+func (m mockEC2Client) CreateVolume(*ec2.CreateVolumeInput) (*ec2.Volume, error) {
+	return m.Vol, nil
+}
```

Comment:
I am not sure what is the purpose of this function here. Ca you explain how is it being used in the test?

Reply:
This is to mock the return value of this interface function. 
type mockEC2Client struct {
	ec2iface.EC2API
	Vol *ec2.Volume
}
CreateVolume() will return what we defined as Vol instead of executing the function that involves cloud provider api calls.

Reply:
Got it. My bad.